document/document_test.go

```diff
@@ -44,6 +44,17 @@ func TestOpen(t *testing.T) {
 	testhelper.CompareZip(t, "simple-1.docx", got.Bytes(), true)
 }
 
+func TestOpenStrict(t *testing.T) {
+	wb, err := document.Open("testdata/strict.docx")
+	if err != nil {
+		t.Errorf("error opening document: %s", err)
+	}
+
+	if err := wb.Validate(); err != nil {
+		t.Errorf("created an invalid document: %s", err)
+	}
```

Comment:
What would be the namespace of that output if written out here?
Would be good to add a check for that just so that it is clear what to expect

Reply:
I just checked, the namespace doesn't change. `Save` func searhes the file name via the `DocType`. These doc types are hardcoded there and they are non-strict. But, since strict and non-strict doc type reference the same file names these files come to the zip package successfully and the doc namespace itself is not being changed. I can modify this test, I just do not fully understand what you would like to see here. I can save the doc after validation to file and what shoud I do next?

Reply:
Save to file and check the namespace of the output file would be good.  Either by loading with the xml library and checking or just searching through the xml contents as a datafile, i.e. opening with the zip reader and checking the xml file. Should not have any instance of "purl.oclc.org".

Reply:
It will have this namespace. The namespace is not being changed during saving. It stays the same

Reply:
I modified the test. Now it opens the strict doc, and resaves it to the new file. The new file should be totally valid non-strict doc. This non-strict doc is being opened and validated (the code is taken from `TestOpen`)